### PR TITLE
Automate release announcements

### PR DESCRIPTION
**Pull request by tas50**
Make the machine do my job.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: .expeditor/announce-release.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -exou pipefail
+
+# Download the release-notes for our specific build
+curl -o release-notes.md "https://packages.chef.io/release-notes/${EXPEDITOR_PRODUCT_KEY}/${EXPEDITOR_VERSION}.md"
+
+topic_title="Chef Infra Client $EXPEDITOR_VERSION Released!"
+topic_body=$(cat <<EOH
+We are delighted to announce the availability of version $EXPEDITOR_VERSION of Chef Chef Infra Client.
```
Comment (tduffield) [inline .expeditor/announce-release.sh:10]: Duplicate "Chef"

**Review by tduffield — CHANGES_REQUESTED** [review]
One small change. Otherwise looks good.

File: .expeditor/publish-release-notes.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Packaging
+
+## Security
+
+EOH
+
+  # Push changes back up to GitHub
+  git add .
+  git commit -m "Release Notes for promoted build $EXPEDITOR_VERSION"
+  git push origin main
```
Comment (tduffield) [inline .expeditor/publish-release-notes.sh:29]: The branch for the Wiki has to be master (for now). The wiki branch is different from the main project branch.

**Review by tduffield — CHANGES_REQUESTED** [review]

File: .expeditor/publish-release-notes.sh (hunk 2 above)
Comment (tas50) [reply .expeditor/publish-release-notes.sh]: Fixed

**Review by tduffield — APPROVED** [review]

**Pull request merged**